README.md

```diff
@@ -9,6 +9,10 @@ Binaryen is a compiler and toolchain infrastructure library for WebAssembly, wri
 
 To provide those capabilities, Binaryen has a simple and flexible API for **representing and processing** WebAssembly modules. The interpreter, validator, pretty-printer, etc. are built on that foundation. The core of this is in [wasm.h](https://github.com/WebAssembly/binaryen/blob/master/src/wasm.h), which contains classes that define a WebAssembly module, and tools to process those. For a simple example of how to use Binaryen, see [test/example/find_div0s.cpp](https://github.com/WebAssembly/binaryen/blob/master/test/example/find_div0s.cpp), which creates a module and then searches it for a specific pattern.
 
+Consult the [contributing instructions](Contributing.md) if you're interested in participating.
+
+Current build status: [![Build Status](https://travis-ci.org/WebAssembly/binaryen.svg?branch=master)](https://travis-ci.org/WebAssembly/binaryen)
```

Comment:
I can do it if you want, but it seems like having the intro first is friendlier?